### PR TITLE
Init App: Admin can create an election.

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

File: client/public/ballot/instructions-fill-oval.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 331.69 178.63">
```
Comment (beausmith) [inline client/public/ballot/instructions-fill-oval.svg:1]: delete this file

File: client/public/ballot/instructions-write-in.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 331.69 107.25">
```
Comment (beausmith) [inline client/public/ballot/instructions-write-in.svg:1]: delete this file

File: client/public/manifest.json [hunk 3]
```diff
@@ -1,25 +1,20 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Election Manager",
```
Comment (beausmith) [inline client/public/manifest.json:2]:
```suggestion
  "short_name": "Mail Ballot Manager",
```

File: client/public/manifest.json [hunk 4]
```diff
@@ -1,25 +1,20 @@
...
+  "name": "VotingWorks Election Manager",
```
Comment (beausmith) [inline client/public/manifest.json:3]:
```suggestion
  "name": "VotingWorks Mail Ballot Manager",
```

File: README.md [hunk 5]
```diff
@@ -93,6 +93,13 @@ This tool **does not** provide any features handled by other offline apps such a
   - Screen Summary:
     - number of voters (matching [insert filter and/or search])
 
+## Election Progress States
+
+- **Configure** with election and voter data
+- **Customize** the content on insert pages
+- **Proof** of Voter Mail Ballot
+- **Track** of Voter Mail Ballots
```
Comment (eventualbuddha) [inline README.md:101]:
```suggestion
- **Track** Voter Mail Ballots
```

File: client/package.json [hunk 6]
```diff
@@ -6,14 +6,32 @@
...
+    "pluralize": "^8.0.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-modal": "^3.11.2",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
-    "typescript": "~3.7.2"
+    "styled-components": "^5.1.0",
+    "typescript": "~3.7.2",
+    "zip-stream": "^3.0.1"
```
Comment (eventualbuddha) [inline client/package.json:34]: This is used for creating zip files on the client. I don't think we need it or its associated `.d.ts` file?

File: client/src/components/BubbleMark.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```
Comment (eventualbuddha) [inline client/src/components/BubbleMark.tsx:1]: This file unused?

File: client/package.json (hunk 6 above)
Comment (beausmith) [reply client/package.json:34]: Yes, will remove. This is the downside of attempting to move fast by cargo copying from `election-manager`.

File: client/src/components/BubbleMark.tsx (hunk 7 above)
Comment (beausmith) [reply client/src/components/BubbleMark.tsx:1]: No. Meant to remove it… but then removed the component that used it first and forgot to come back to remove it. 🤦

File: .circleci/config.yml [hunk 8]
```diff
@@ -12,6 +12,14 @@ jobs:
...
     steps:
       - checkout
+      - run:
+          name: "Update Node.js and npm"
+          command: |
+            curl -sSL "https://nodejs.org/dist/v12.10.0/node-v12.10.0-linux-x64.tar.xz" | sudo tar --strip-components=2 -xJ -C /usr/local/bin/ node-v12.10.0-linux-x64/bin/node
+            curl https://www.npmjs.com/install.sh | sudo bash
+      - run:
+          name: Check current version of node
+          command: node -v
```
[the remark this answers is not in the document]
Comment (beausmith) [reply .circleci/config.yml:22]: Can we cache things?

[the remark this answers is not in the document]
Comment (eventualbuddha) [reply .circleci/config.yml]: If we're already installing Chrome ourselves, why don't we just use a docker image with the node version we want already installed? We don't even have any "browser" tests, do we?

**Pull request merged**